### ISSUE TITLE
#PAGIRINYA REPAIR CAFE TEAM.

### ISSUE THREAD

**Issue by maliarich** [ASKnet-Open-Training/Minimal-Repair-Setup#5]
@gibertgibs.

**Comment by gibertgibs** [comment]
Perfect...Okay fine

**Comment by gibertgibs** [comment]
Pagirinya Team set for collaboration.

**Comment by timmwille** [comment]
> Pagirinya Team set for collaboration.

share some setup impressions

**Comment by timmwille** [comment]
Any challenges so far where we might be able to help with?

**Comment by gibertgibs** [comment]
# Pagirinya Remote Repair Cafe 
*The receptionists*
1. Akbar Ismail
2. Lulujoa Rebecca
The two personels served the responsibility of welcoming and registering all the devices that were to be worked upon.

![IMG_20210216_145757_901](https://user-images.githubusercontent.com/69734297/108060190-9b0a0680-7067-11eb-9647-2d5f9a42cf6a.jpg)

*The repair Technicians*

*Table one: Radio Repair*
Table one was strictly meant for radio repairs and the technicians responsible were;
1. Wafela Andrew
2. Acidri Tairi

*Table two: Phone repairs*
Table two was also strictly for phone repairs. And the table was strictly for phone repairs.
1. Kenyi Jimmy Loku
2. Ijjo Thomas William

*Table three: Inverters, and solar lamp repairs*
Table three was also meant for inverters and solar repairs and gladly the protuccal was well observed. The technicians were;
1. Emmanuel Iceku
2. Makumai Friday Game
3. Eriku Christopher

**Comment by chardso** [comment]
Inverters,solar lamps were also handled by table one.it was actually the multipurpose table.

**Comment by chardso** [comment]
Am not seing the camera in the list and yet it was Registered and fixed.

**Comment by chardso** [comment]
We graded technicians according to their experiences in the field of technicalities.

**Comment by timmwille** [comment]
> We graded technicians according to their experiences in the field of technicalities.

Nice this one sounds interesting, maybe explain in more detail?

**Comment by vugawilliam** [comment]
I have not seen camera repaired on the list.

**Comment by vugawilliam** [comment]
However! They are communities members who came later during the session, due to time factor we haven't attend them and there asked weather we could have such lovely services again.

**Comment by vugawilliam** [comment]
Yes, we start our off-line repairs by 10:00am, why as team leader I though wise to start early so that we can repair many devices rather then starting 3:00pm office time. By the time reaching at offical time we move in online although the network is not so challengeing to us on that day.

**Comment by chardso** [comment]
![Uploading IMG_20211030_205338_311.jpg…]()
Need of toolkit pic.

**Comment by timmwille** [comment]
Ah nice, yes the documentation in general is good, though a Tools List would really help, including pictures or is the list complete? where is the kit now @PyGamma @gibertgibs ?
